src/main/java/com/alkemy/ong/web/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.alkemy.ong.web.controllers;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+        @ExceptionHandler(value
+                = { IllegalArgumentException.class, IllegalStateException.class })
+        protected ResponseEntity<Object> handleConflict(
+                RuntimeException ex, WebRequest request) {
+            String bodyOfResponse = "El id no existe";
```

Comment:
not spanish! igual podrías usar el mensaje que viene de la excepción q ya lo seteas al mensaje en el gateway